lib/shopify_api/resources/fulfillment_order_fulfillment.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyAPI
+  class FulfillmentOrderFulfillment < Base
```

Comment:
What does this class represent? Can we return `Fulfillment` objects in `FulfillmentOrder#fulfillments` instead?

Reply:
Good question--being discussed in comment thread immediately below.

Reply:
We can either use the same `Fulfillment` object or a `FulfillmentV2` if we want to make a clear distinction between legacy and the newer fulfillment resource type. I'm starting to think using a single `Fulfillment` resource and documenting operations as legacy/new might be the way to go.